src/main.rs

```diff
@@ -31,29 +31,74 @@ fn try_reload_config(config: &mut Config, config_file: &String) {
     }
 }
 
+///// Start parsing client input
+
 fn _parse_client_cmd(raw: &String) -> Result<String> {
     if let Some(cmd) = raw.split_whitespace().next() {
+        Ok(cmd.to_string())
+    } else {
+        Err(Error::WrongClientInputFormat)
+	}
+}
+
+fn _parse_client_arg(raw: &String) -> Result<String> {
```

Comment:
Why underscore in the fonction name ?
Couldn't the function name be more self explanatory ?
ex: parse_job_name_from_client_input

Reply:
it was to follow your notation in config.rs

Reply:
I use underscore for private function for a struct, this funtion is not private and not in a struct :eyes: